lua/ulx/modules/sh/cfc_checklua.lua

```diff
@@ -0,0 +1,41 @@
+CFCUlxCommands.checklua = CFCUlxCommands.checklua or {}
+local cmd = CFCUlxCommands.checklua
+
+CATEGORY_NAME = "Utility"
+
+if SERVER then
+    util.AddNetworkString( "CFC_ULX_StatCheckCL" )
+    util.AddNetworkString( "CFC_ULX_StatCheckSV" )
+end
+
+local awaitingResponse = {}
+
+net.Receive( "CFC_ULX_StatCheckSV", function( _, ply )
+    if awaitingResponse[ply] == nil then return end
+    local convar = net.ReadBool()
+    ulx.fancyLogAdmin( awaitingResponse[ply], true, "#T's sv_allowcslua value is " .. tostring( convar ), ply )
```

Comment:
You should be able to get away without using `tostring` here, as using string concatenation should do that for you.

```suggestion
    ulx.fancyLogAdmin( awaitingResponse[ply], true, "#T's sv_allowcslua value is " .. convar, ply )
```